### PR TITLE
Update to allowed ports

### PR DESCRIPTION
**Pull request by stevedicko**
Updated the Patching and MMaaS FAQs to correctly state the firewall ports that need to be open as part of the Prereqs sections

### CODE REVIEW

File: articles/managed-services/man-monitoring-faq.md [hunk 1]
```diff
@@ -31,9 +31,24 @@ UKCloud uses the same technologies we use in house to monitor events across our
...
 
-## Do I need to install an agent within my VM to enable this service?
+## Prerequisites
 
-By default, UKCloud will provide this service without the use of an agent by utilising SNMP traps. In situations where non-SNMP events need to be collected, UKCloud may opt to provide a dedicated Monitoring Agent, which can be deployed into the customer's VM to provide a more advanced method of event collection. To ensure the correct operation of the Monitoring Agent once installed, port 5666 will need to be opened on Linux VMs, and port 5985 and/or port 5986 must be opened for Windows VMs.
+### Do I need to install an agent within my VM to enable this service?
+
+By default, UKCloud will provide this service without the use of an agent by utilising SNMP traps. In situations where non-SNMP events need to be collected, UKCloud may opt to provide a dedicated Monitoring Agent, which can be deployed into the customer's VM to provide a more advanced method of event collection.
+
+### Do I need to open any specific firewalls ports or allow firewall rules?
```
Comment (SteveHallUKCLoud) [inline articles/managed-services/man-monitoring-faq.md:40]: Firewalls or Firewall?

**Review by SteveHallUKCLoud — CHANGES_REQUESTED** [review]
Just one comment

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

**Review by suehighmoor — APPROVED** [review]
LGTM

File: articles/managed-services/man-monitoring-faq.md (hunk 1 above)
Comment (stevedicko) [reply articles/managed-services/man-monitoring-faq.md]: My bad, should be the singular Firewall - good spot!

**Review by stevedicko — COMMENTED** [review]
Agree to the addition of pre-reqs section

**Pull request merged**